api/proto/metrics/metrics.proto

```diff
@@ -144,6 +145,27 @@ message appMetric {
   repeated appDiskMetric disk = 6;
 }
 
+message logMetric {
+  uint64 numDeviceLogsSent = 1;
```

Comment:
The struct is a LogEvent, right? So should we call this Events instead of Logs?

Reply:
Ok I'll rename them

Reply:
done